src/main/java/org/springframework/samples/petclinic/rest/importcsv/ImportCSV.java

```diff
@@ -1,9 +1,11 @@
 package org.springframework.samples.petclinic.rest.importcsv;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
```

Comment:
I think it was said that no new dependencies could be added

Reply:
This is not a new dependency. It's a file/class from an existing dependency.